OrbitCore/Path.cpp

```diff
@@ -97,13 +92,10 @@ bool Path::DirExists(const std::string& dir) {
 }
 
 std::string Path::GetBasePath() {
```

Comment:
this is used only Dllpaht which is not used anywhere else - remove this and GetDllPath and base_path_

Reply:
this result then in Path not having any state anymore and only static functions. You can then consider changing it from a class to a namespace Path. (I have a slight preference towards namespace instead static class, but do not care to much)

Reply:
True, I changed it to a namespace now, thanks